kyo-core/shared/src/main/scala/kyo/Console.scala

```diff
@@ -19,34 +19,41 @@ final case class Console(unsafe: Console.Unsafe):
       * @param s
```

Comment:
Can you also update the top `Console` scaladoc with the rationale for not having a pending abort + the role of `checkErrors`? I think it's important to clearly flag this behavior to the user.

Reply:
Done. Let me know if the text works for you.

Reply:
perfect! thanks